client/post-editor/editor-seo/style.scss

```diff
@@ -0,0 +1,11 @@
+.editor-drawer__editor-seo {
+	.token-field__remove-token {
+		display: none;
+	}
+
+	.token-field__token-text {
+		border-radius: 4px;
+		background-color: lighten( $gray, 10% );
+		padding: 0 6px;
+	}
+}
```

Comment:
If your editor supports it, you should download an [EditorConfig plugin](http://editorconfig.org/). All files should [end in a newline](https://github.com/Automattic/wp-calypso/blob/07bb4c459178d8aed5be97de6ae9072f2c960383/.editorconfig#L9).